packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -396,6 +416,24 @@ class _DraggableState<T> extends State<Draggable<T>> {
   }
 }
 
+/// Represents the draggable's current details, such as whether the
+/// [DragTarget] accepted it, its [Velocity], and its [Offset].
+class DraggableDetails {
+  DraggableDetails._({
+    this.wasAccepted,
+    this.velocity,
+    this.offset});
```

Comment:
trivial nit: flutter style would put the `});` on the next line, vertically aligned with the constructor name.

Reply:
Consider it done.